curp/src/server/cmd_worker/conflict_checked_mpmc.rs

```diff
@@ -190,8 +191,8 @@ enum OnceState {
 /// Internally it maintains a dependency graph of conflicting cmds
 
 struct Filter<C: Command, CE> {
-    /// Index from `ProposeId` to `vertex`
-    cmd_vid: HashMap<ProposeId, u64>,
+    /// Index from `LogIndex` to `vertex`
+    cmd_vid: HashMap<LogIndex, u64>,
```

Comment:
We'd better rename this `cmd_vid` because it implicitly indicates the mapping relationship between a command id and a vertex id. But now this assumption is not valid. How about `log_vid` or `entry_vid`?

Reply:
Renamed